.github/workflows/ci.yml

```
@@ -10,142 +7,146 @@ on:
+          name: Espressif-IDE-${{ env.VERSION }}-macosx.cocoa.x86_64
+          path: releng/ide-dmg-builder/Espressif-IDE-macosx-cocoa-x86_64.dmg
+
+      - name: Upload espressif-ide-macosx.cocoa.aarch64 dmg
+        if: ${{ !cancelled() }}
+        uses: actions/upload-artifact@v4
+        with:
+          name: Espressif-IDE-${{ env.VERSION }}-macosx.cocoa.aarch64
+          path: releng/ide-dmg-builder/Espressif-IDE-macosx-cocoa-aarch64.dmg
+
+  build_and_test:
+	strategy:
```

actionlint (1.7.7)

could not parse as YAML: yaml: line 107: found character that cannot start any token

(syntax-check)